modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -2,9 +2,44 @@
 
 MuleSoft captures usage data for Mule flows, Mule messages, and data throughput, but not all of this data is aggregated in usage reports. 
 
-
 For select customers in the US control plane, MuleSoft offers a pricing and packaging model for Anypoint Platform that allots a number of Mule flows, Mule messages, and data throughput (measured as network I/O bytes). Not all types of Mule flows and messages count toward the allotments in a package. 
 
```

Comment:
Without headers, the table descriptions look a bit confusing because first it defines what the metric represents, and then, how it is counted towards usage. 

I'd suggest either adding headers like **Product | Metric description | Consumption details** (with better terms maybe), or perhaps an introductory sentence before the table: 
"The following table shows the metrics measured for each MuleSoft product and how their consumption is accounted."